app/renderer/components/main/braveryPanel.js

```diff
@@ -560,26 +558,19 @@ class BraveryPanel extends React.Component {
                   </BraveryPanelDropdown>
                 </div>
 
-                <div data-l10n-id='fingerprintingProtection' className={css(
+                <div className={css(
                   !this.props.shieldsUp && styles.braveryPanel__body__advanced__control__forms__title_disabled,
                   !this.props.isCompactBraveryPanel && gridStyles.row3col2,
-                  !this.props.isCompactBraveryPanel && styles.braveryPanel__body__advanced__control__forms__title,
                   this.props.isCompactBraveryPanel && gridStyles.row5col1,
-                  this.props.isCompactBraveryPanel && styles.braveryPanel_compact__body__advanced__control__forms__title
-                )} />
-                <span className={cx({
-                  [css(gridStyles.row3col2)]: !this.props.isCompactBraveryPanel,
-                  [css(gridStyles.row5col1)]: this.props.isCompactBraveryPanel,
-                  [css(styles.braveryPanel__body__advanced__control__forms__title_disabled)]: !this.props.shieldsUp,
-                  [css(styles.braveryPanel_compact__body__advanced__control__forms__title)]: this.props.isCompactBraveryPanel,
-                  [css(styles.braveryPanel__body__advanced__control__forms__title)]: !this.props.isCompactBraveryPanel,
-                  fa: true,
-                  pullRight: true,
-                  'fa-question-circle': true
-                })}
-                  title={config.fingerprintingInfoUrl}
-                  onClick={this.onInfoClick}
-                />
+                  styles.braveryPanel__body__advanced__control__fpWrapper,
+                  styles.braveryPanel__body__advanced__control__forms__title
+                )}>
+                  <span data-l10n-id='fingerprintingProtection' />
+                  <span className={globalStyles.appIcons.question}
```

Comment:
`braveryPanel__body__advanced__control__fpWrapper ` is cleated to wrap the l10n string and the question mark to align them with flex.